corehq/apps/accounting/utils.py

```diff
@@ -116,10 +116,9 @@ def domain_has_privilege(domain, privilege_slug, **assignment):
     from corehq.apps.accounting.models import Subscription
     try:
         plan_version = Subscription.get_subscribed_plan_by_domain(domain)[0]
```

Comment:
That's a good point. The [plan version is cached on the request](https://github.com/dimagi/commcare-hq/blob/abcd219fe4d88e974e971220b7da244bc98ec0d1/corehq/apps/domain/middleware.py#L38). Unfortunately it looks like this function does not have easy access to the request. Maybe we should change that? I'd prefer to do that in a separate PR.